superset/charts/schemas.py

```diff
@@ -1002,12 +1002,12 @@ class Meta:  # pylint: disable=too-few-public-methods
         allow_none=True,
     )
     filters = fields.List(fields.Nested(ChartDataFilterSchema), allow_none=True)
-    granularity = fields.String(
+    granularity = fields.Raw(
```

Comment:
@diegomedina248, The granularity field has deprecated since the Druid connector has been removed by [PR](https://github.com/apache/superset/pull/20338).

Reply:
Doesn't this apply to other engines? I think I tested with postgres at the time